packages/core/core-flows/src/cart/workflows/complete-cart.ts

```diff
@@ -330,6 +336,11 @@ export const completeCartWorkflow = createWorkflow(
 
       registerUsageStep(promotionUsage)
 
+      afterOrderCreated = createHook("afterOrderCreated", {
```

Comment:
thought: Should we drop the prepositions here to be consistent with other hooks, e.g. `cartCreated`?